.ci.yaml

```diff
@@ -4272,7 +4272,7 @@ targets:
     properties:
       tags: >
         ["devicelab", "ios", "mac"]
-      task_name: external_ui_integration_test_ios
+      task_name: external_textures_integration_test_ios
```

Comment:
This will affect the benchmark metrics as they are tied to task_name. If this is the way to go, you may want to give engine people a heads up. /cc @zanderso

Reply:
These aren't benchmarks confusingly.